### PR TITLE
[bitnami/mariadb-galera] Add JSON Schema

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

**Description of the change**

This PR includes a basic JSON schema for the MariaDB Galera chart. It doesn't cover all the possible values of the chart but just the ones that, in my opinion, are more likely to be modified.

This is valid for Helm 3 to validate some values and, at the same time, for Kubeapps to render a simple form so the chart containing it is easier to configure and deploy. See https://github.com/kubeapps/kubeapps/blob/master/docs/developer/basic-form-support.md for more information.

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/mariadb-galera/values-production.yaml [hunk 1]
```diff
@@ -186,10 +186,10 @@ ldap:
   ## Enable LDAP support
   ##
   enabled: false
-  uri:
-  base:
-  binddn:
-  bindpw:
+  uri: ""
```
Comment (andresmgot) [inline bitnami/mariadb-galera/values-production.yaml:189]: I'm curious, is there a linter or something for things like this?

File: bitnami/mariadb-galera/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+      "form": true,
+      "properties": {
+        "create": {
+          "type": "boolean",
+          "form": true,
+          "title": "Create RBAC rules",
+          "description": "Specify whether RBAC resources should be created and used"
+        }
+      }
+    },
```
Comment (andresmgot) [inline bitnami/mariadb-galera/values.schema.json:163]: is this worth a parameter? I don't think disabling RBAC is a common use case

Comment (juan131) [reply bitnami/mariadb-galera/values.schema.json:163]: it's disabled by default. This parameter allows you enabling it

File: bitnami/mariadb-galera/values-production.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/mariadb-galera/values-production.yaml:189]: This specific behaviour escapes linter's control

File: bitnami/mariadb-galera/values.schema.json (hunk 2 above)
Comment (andresmgot) [reply bitnami/mariadb-galera/values.schema.json:163]: mmh, by default this chart doesn't work with a cluster with RBAC enabled?

**Pull request merged**